app/Relationship.php

```diff
@@ -6,5 +6,13 @@
 
 class Relationship extends Model
 {
-    //
+    public function activities()
+    {
+        return $this->morphMany('App\Activity', 'activity');
+    }
+
+    public function following_user()
```

Comment:
Please use camelCase() for methods as mention on the PSR-1 (standards for php code)
https://www.php-fig.org/psr/psr-1/#43-methods

Reply:
I modified function name `following_user` to `followingUser` on these file.
- `app\Relationship.php`
- `resources\views\components\user-activity.blade.php`